app/assets/javascripts/modules/inline-image-modal.js

```diff
@@ -127,7 +129,7 @@ InlineImageModal.prototype.performAction = function (item) {
 }
 
 InlineImageModal.prototype.initComponents = function () {
-  window.GOVUK.modules.start()
+  window.GOVUK.modules.start($(this.$module))
```

Comment:
minor, but I think given we have a convention for $ on variables the usage of jQuery might be clearer by using the jQuery global variable rather than $

Reply:
@alex-ju what do you think? I was going from the use of `$` in govuk_frontend_toolkit, which I've used in the tests. Can change everywhere if we agree it's preferable :-).

Reply:
I see @kevindew's point that the use of `$` for illustrating `nodeList` objects is not great as it causes confusion with the jQuery alias, but on the other hand we use it this way all over `govuk_publishing_components` and other projects across GOVUK. I hope to get rid of jQuery from this project soon so I have no strong opinions about how it will looks like until then.